src/test/a11y/tests/system-admin-routes.test.ts

```diff
@@ -125,4 +125,38 @@ describe('Accessibility - System Admin Routes', () => {
             testAccessibility(route.path, route.parameter);
         });
     });
+
+    describe('Page with Errors', () => {
+        describe('Create System Admin Account Page', () => {
+            const url = '/create-system-admin-account';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, {'create-system-admin-account': ''});
+            });
+        });
+
+        describe('Reference Data upload Page', () => {
+            const url = '/reference-data-upload';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, {});
+            });
+        });
+
+        describe('Delete Court Reference Data Confirmation Page', () => {
+            const url = '/delete-court-reference-data-confirmation';
+
+            describe('with invalid input data', () => {
+                testAccessibility(url, '', true, {'delete-choice': null});
+            });
+        });
+
+        describe('Bulk Create Media Accounts Page', () => {
+            const url = '/bulk-create-media-accounts';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, {});
```

Comment:
{} is the default value for testAccessibility(). No need to pass in the default value